### PR TITLE
Feat/rxdb

### PR DESCRIPTION
**Pull request by pChausseC**
- replace pouchdb with rxdb and rxdb-hooks

### CODE REVIEW

File: client/src/providers/rxdb.provider.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+export const RxdbProvider = (props: { children: React.ReactNode }) => {
+  const [db, setDb] = React.useState<RxDatabase | undefined>(undefined);
+
+  React.useEffect(() => {
+    // RxDB instantiation can be asynchronous
+    initialize()
+      .then(setDb)
+      .catch(() => {
+        //ignore
```
Comment (mrabdibdi-anvil) [inline client/src/providers/rxdb.provider.tsx:16]: what happens when there is no DB to handle the data ?

File: client/pnpm-lock.yaml
Comment (mrabdibdi-anvil) [inline client/pnpm-lock.yaml:1]: can we remove pnpm completely ?

File: client/src/components/clear-project-button.tsx [hunk 2]
```diff
@@ -1,32 +1,25 @@
...
-      .then(({ data }) => {
-        if (data) {
-          setShow(true);
-        }
-      })
-      .catch(() => {
-        //ignore
-      });
-  }, []);
+  const { result } = useRxData<ActiveProject>("activeProject", (collection) =>
```
Comment (mrabdibdi-anvil) [inline client/src/components/clear-project-button.tsx:11]: should it be async ? can we press the button before having the response from the local DB ?

**Review by mrabdibdi-anvil — APPROVED** [review]
few non blocking questions

File: client/src/components/clear-project-button.tsx (hunk 2 above)
Comment (pChausseC) [reply client/src/components/clear-project-button.tsx:11]: 1. no, this is a hook. 2. look a few lines down and you will see the button is not rendered until result is defined.

File: client/pnpm-lock.yaml
Comment (pChausseC) [reply client/pnpm-lock.yaml:1]: yeah sorry I used it by reflex

File: client/src/providers/rxdb.provider.tsx (hunk 1 above)
Comment (pChausseC) [reply client/src/providers/rxdb.provider.tsx:16]: you cant upload a file

**Review by JFKFred — APPROVED** [review]

**Pull request merged**